test/snapshot/__snapshots__/list.test.js.snap

```diff
@@ -1630,18 +1745,18 @@ Program {
                     },
                   },
                 ],
-                "kind": "array",
+                "kind": "list",
                 "loc": Location {
                   "end": Position {
-                    "column": 19,
+                    "column": 24,
                     "line": 1,
-                    "offset": 19,
+                    "offset": 24,
```

Comment:
I'm not sure why the locations are different.

Reply:
see comment on locations